src/Types/Input/InputType.hack

```diff
@@ -2,15 +2,21 @@ namespace Slack\GraphQL\Types;
 
 use namespace HH\Lib\C;
 use namespace Slack\GraphQL;
-use namespace Graphpinator\Parser\Value;
+use namespace Graphpinator\Parser\{TypeRef, Value};
 
 /**
  * GraphQL type that may be used for field/directive arguments and for variable values.
  * Includes scalar types, enums and input objects (GraphQL's version of dict).
  *
  * @see https://spec.graphql.org/draft/#sec-Input-and-Output-Types
  */
-abstract class InputType<TCoerced> extends BaseType {
+abstract class InputType<+TCoerced> extends BaseType {
+
+    const dict<string, classname<NamedInputType>> BUILTINS = dict[
+        IntInputType::NAME => IntInputType::class,
+        StringInputType::NAME => StringInputType::class,
+        BooleanInputType::NAME => BooleanInputType::class,
```

Comment:
Would it make sense to add these to the schema's `INPUT_TYPES`? They should be surfaced during introspection, right?

Reply:
Good idea, I'll move it.